pro/src/pages/Signup/SignupForm/SignupForm.tsx

```diff
@@ -86,7 +91,16 @@ const SignupForm = (): JSX.Element => {
     setShowAnonymousBanner(false)
     const response = await getSirenDataAdapter(siren)
     if (response.isOk) {
-      formik.setFieldValue('legalUnitValues', response.payload.values)
+      let values = undefined
+      if (response.payload.values != null) {
+        const { apeCode, ...rest } = response.payload.values
```

Comment:
rest pour "se reposer" en anglais ?
```suggestion
        const { apeCode, ..._sirenData } = response.payload.values
```

Reply:
voir meme 
```suggestion
        const { apeCode,  } = response.payload.values
```

Reply:
Non, rest pour ["the rest"](https://dictionary.cambridge.org/dictionary/english/rest)

Reply:
Et pour la deuxième suggestion, je veux récupérer tous les champs de mon objet sans `apeCode`.